---
geometry_msgs/PoseStamped goal_pose